controllers/apply/form-router-next/submission.js

```diff
@@ -103,26 +103,39 @@ module.exports = function(
                     formId: formId
                 });
 
-                const contentVersionPromises = form
-                    .getCurrentFields()
-                    .filter(field => field.type === 'file')
-                    .map(async function(field) {
-                        return buildMultipartData({
-                            formId: formId,
-                            applicationId: currentApplication.id,
-                            filename: field.value.filename
-                        }).then(versionData => {
-                            return salesforce.contentVersion({
-                                recordId: salesforceRecordId,
-                                attachmentName: `${field.name}${path.extname(
-                                    field.value.filename
-                                )}`,
-                                versionData: versionData
+                /**
+                 * We can consider an application "submitted" once the
+                 * main FormData record has been created.
+                 *
+                 * Consider ContentVersion step optional
+                 * but log an error for investigation if upload fails.
+                 */
+                try {
+                    const contentVersionPromises = form
+                        .getCurrentFields()
+                        .filter(field => field.type === 'file')
+                        .map(async function(field) {
+                            return buildMultipartData({
+                                formId: formId,
+                                applicationId: currentApplication.id,
+                                filename: field.value.filename
+                            }).then(versionData => {
+                                return salesforce.contentsVersion({
+                                    recordId: salesforceRecordId,
+                                    attachmentName: `${
+                                        field.name
+                                    }${path.extname(field.value.filename)}`,
+                                    versionData: versionData
+                                });
                             });
                         });
-                    });
 
-                await Promise.all(contentVersionPromises);
+                    await Promise.all(contentVersionPromises);
+                } catch (error) {
+                    logger.error('Error creating ContentVersion', error, {
+                        formId: formId
```

Comment:
Worth logging the application ID here too to help us investigate? With the recent one I had to just look for submission timestamps which matched the log event date.